apps/cowswap-frontend/src/modules/account/hooks/useCloseAccountModalOnNavigate.ts

```diff
@@ -0,0 +1,18 @@
+import { useLayoutEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+import { useCloseAccountModal } from './useToggleAccountModal'
+
+/**
+ * Every time the url changes, closes the account modal
+ * If it's not open it has no effect
+ */
+export function useCloseAccountModalOnNavigate(): null {
+  const location = useLocation()
+  const closeAccountModal = useCloseAccountModal()
+
+  useLayoutEffect(() => {
+    closeAccountModal()
+  }, [location.pathname])
```

Comment:
Missing hook dependency `closeAccountModal`

Reply:
I can't survive without the automatic linter